my-app/src/views/Main/Main.jsx

```diff
@@ -7,6 +7,9 @@ import { connect} from "react-redux";
 import { updateValue } from "../../store/actions";
 import "./Main.css";
 import {Link, useLocation} from "react-router-dom";
+import { useHistory } from 'react-router-dom';
```

Comment:
![image](https://github.com/Arjun-Girish/FYP-InteractiveCourseMapper/assets/87962390/bd7639f0-3773-4042-8f52-24f64df809e3)
Move back button to the bottom of the screen, background of the button needs to be white with blue outline

Change position of legend
![Uploading image.png…]()